content/02.intro.md

```diff
@@ -1,3 +1,16 @@
 ## Introduction {.page_break_before}
 
-This is an introductory section...
\ No newline at end of file
+Machine learning is gaining momentum in analysis of biomedical data as the data collection leans increasingly towards high dimensionality data and the algorithmic applications become increasingly transparent and interpretable. 
+
+Application of machine learning to any dataset is complicated but the complexity of applying these to biomedical dataset is enormous. 
+The proper application and interpretation requires depth of knowledge not only in the biomedical domain but also a clear understanding of the methods and the underlying assumptions of each of the methods.
+
+Rare disease has not benefitted significantly from machine learning applications yet due to various reasons including lack of statistical power in dataset size, heterogeneity in available data, and sensitivity of machine learning methods to misinterpretation in view of small datasets. 
+However recent advances in the methodologies to accomodate rarity of samples and increased transparency in model outputs have encouraged application of machine learning in rare disease.
+We anticipate higher occurrence of such applications in the near future and aim to highlight the current state-of-art in this perspective.
+
+Application of machine learning to any kind of data consists of the following major steps: (1) data evaluation and question formulation, (2) selection of normalization/dimension reduction to mitigate technical differences, (3) selection of appropriate algorithms to answer the formulated question, (4) evaluation of the answers generated by the algorithm. 
```

Comment:
`dimension reduction to mitigate technical differences` Not sure about this part specifically. Dimension reduction could be for feature learning. Can you tell me more about what you meant?

Reply:
Fair point. In this instance, I was referring mostly to exploratory dimension reduction analyses like PCA, tSNE etc which are used to explore existing groups in the data (to test if they are technical or disease-related) and are used in conjunction with normalization techniques. 

But important to state the feature learning side of dimension reduction as well. In this case, it was implied in the selection of algorithm point (where various dimension reduction steps may be used for clustering/classifying data). I can add that more explicitly.